docs/resources/Guild.md

```diff
@@ -687,6 +687,17 @@ This endpoint takes a JSON array of parameters in the following format:
 | lock_permissions | ?boolean   | syncs the permission overwrites with the new parent, if moving to a new category |
 | parent_id        | ?snowflake | the new parent ID for the channel that is moved                                  |
 
+## List Active Threads % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/threads/active
+
+Returns all active threads in the guild, including public and private threads. Threads are ordered by their `id`, in descending order.
+
+###### Response Body
+
+| Field    | Type                                                                            | Description                                                                                  |
+|----------|---------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| threads  | array of [channel](#DOCS_RESOURCES_CHANNEL/channel-object) objects              | the active threads                                                                           |
+| members  | array of [thread members](#DOCS_RESOURCES_CHANNEL/thread-member-object) objects | a thread member object for each returned thread the current user has joined                  |
+
```

Comment:
What is with 
| has_more | boolean                                                                         | whether there are potentially additional threads that could be returned on a subsequent call |

Reply:
Yeah, it was removed, it was confusing because it was always true
And we don't have an efficient way of paginating here and it's pretty cheap to just return everything since there is an active thread cap

Reply:
That means we get every thread till the cap?

Reply:
What is the internal cap?
I'm interested. Would like to set a note in the docs.

Reply:
I replied to this in DDevs a couple of times search for "1k" from me there. No plans to document the limit, since it's certainly possible it could change early on.

Reply:
Oky doki.
Thanks.